assets/src/edit-story/components/canvas/canvasProvider.js

```diff
@@ -48,6 +48,7 @@ function CanvasProvider({ children }) {
     setEditingElementWithState,
     setEditingElementWithoutState,
     clearEditing,
+    getNodeForElement,
```

Comment:
I **hate** how big this component is! I think it's our worst one in the entire codebase. Buuuuut that's outside the scope of this ticket.

Reply:
Agree. Definitely unnecessarily large.